v1/topdown/graphql_bench_test.go

```diff
@@ -0,0 +1,363 @@
+// Copyright 2025 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package topdown
+
+import (
+	"context"
+	_ "embed"
+	"testing"
+
+	"github.com/open-policy-agent/opa/v1/ast"
+	"github.com/open-policy-agent/opa/v1/topdown/cache"
+)
+
+func BenchmarkGraphQLSchemaIsValid(b *testing.B) {
+
+	// Share an InterQueryValueCache across multiple runs
+	// Tune number of entries to exceed number of distinct GQL schemas
+	in := `{"inter_query_builtin_value_cache": {"max_num_entries": 10},}`
+	config, _ := cache.ParseCachingConfig([]byte(in))
+	valueCache := cache.NewInterQueryValueCache(context.Background(), config)
+
+	benches := []struct {
+		desc   string
+		schema *ast.Term
+		cache  cache.InterQueryValueCache
+		result *ast.Term
+	}{
+		{"Trivial Schema - string", ast.NewTerm(ast.String(employeeGQLSchema)), nil, ast.BooleanTerm(true)},
+		{"Trivial Schema with cache - string", ast.NewTerm(ast.String(employeeGQLSchema)), valueCache, ast.BooleanTerm(true)},
+		{"Trivial Schema - AST object", ast.NewTerm(ast.MustParseTerm(employeeGQLSchemaAST).Value.(ast.Object)), nil, ast.BooleanTerm(true)},
+		{"Trivial Schema with cache - AST object", ast.NewTerm(ast.MustParseTerm(employeeGQLSchemaAST).Value.(ast.Object)), valueCache, ast.BooleanTerm(true)},
+	}
```

Comment:
For readability, it'd help if these were multi-line, as in your other bench tests.